test/e2e/network/dns.go

```diff
@@ -219,7 +219,7 @@ var _ = SIGDescribe("DNS", func() {
 		validateDNSResults(f, pod, append(wheezyFileNames, jessieFileNames...))
 	})
 
-	It("should provide DNS for pods for Hostname and Subdomain", func() {
+	It("should provide DNS for pods for Hostname and Subdomain [LinuxOnly]", func() {
```

Comment:
Just an FYI, the only LinuxOnly part of this test is the Hostname part, which expects the /etc/hosts file to be managed by Kubelet, and which is not currently possible on Windows, as we cannot mount individual files to containers.

I have a PR which splits this test into 2 parts: https://github.com/kubernetes/kubernetes/pull/75591

Reply:
Yes, let's not drop adding [LinuxOnly] tag here. Definitely #75591 would override this e2e over a period of time.

Reply:
Done. Included as a comment.